packages/aws-cdk-lib/aws-lambda/lib/lambda-insights.ts

```diff
@@ -112,16 +113,16 @@ export abstract class LambdaInsightsVersion {
         produce: (context) => getVersionArn(context.scope, insightsVersion),
       });
 
-      public _bind(_scope: Construct, _function: IFunction): InsightsBindConfig {
-        const arch = _function.architecture?.name ?? Architecture.X86_64.name;
+      public _bind(scope: Construct, fn: IFunction): InsightsBindConfig {
+        const arch = fn.architecture?.name ?? Architecture.X86_64.name;
```

Comment:
who decided it made sense to continue to use `_function` 😂

Reply:
🤷🏻